ui/src/components/integrationFormFields/integrationFormFields.jsx

```diff
@@ -18,7 +19,7 @@ export const IntegrationFormFields = (props) => {
   } = extensionProps;
   useEffect(() => {
     initialize(initialData);
-  }, []);
+  }, [initialData, initialize]);
```

Comment:
Pls remove the deps.
It is intended to be executed only on component mount phase.